legal-test-fixture/legal_api/api/converter/ExcelWriter.py

```diff
@@ -0,0 +1,280 @@
+        self.__business_sheet.write(
+            self.__business_sheet_row_index, 4, format_date(business.dissolution_date))
+        self.__business_sheet.write(
+            self.__business_sheet_row_index, 5, format_date(business.last_ar_date))
+        self.__business_sheet.write(
+            self.__business_sheet_row_index, 6, format_date(business.last_agm_date))
+        self.__business_sheet.write(
+            self.__business_sheet_row_index, 7, format_date(business.fiscal_year_end_date))
+        self.__business_sheet.write(
+            self.__business_sheet_row_index, 8, format_non_date(business.tax_id))
+        self.__business_sheet.write(
+            self.__business_sheet_row_index, 9, format_non_date(business.last_ledger_id))
+        self.__business_sheet.write(self.__business_sheet_row_index, 10, format_non_date(
+            business.last_remote_ledger_id))
+        self.__business_sheet.write(self.__business_sheet_row_index, 11, format_date(
+            business.last_ledger_timestamp))
+        self.__business_sheet.write(
+            self.__business_sheet_row_index, 12, format_date(business.last_modified))
+
+        self.__business_sheet_row_index += 1
+
+        directors = business.directors.all()
+        for director in directors:
+            self.__write_director_to_excel(business.identifier, director)
+
+        delivery_address = business.delivery_address.one_or_none()
+        if (delivery_address):
+            self.__write_business_address_to_excel(
+                business.identifier, delivery_address)
+
+        mailing_address = business.mailing_address.one_or_none()
+        if (mailing_address):
+            self.__write_business_address_to_excel(
+                business.identifier, mailing_address)
+
+        filings = business.filings.all()
+        for filing in filings:
+            self.__write_filing_to_excel(business.identifier, filing)
+
+    def __write_director_to_excel(self, business_identifier, director):
+        self.__director_sheet.write(
+            self.__director_sheet_row_index, 0, format_non_date(business_identifier))
+        self.__director_sheet.write(
+            self.__director_sheet_row_index, 1, format_non_date(director.first_name))
+        self.__director_sheet.write(
+            self.__director_sheet_row_index, 2, format_non_date(director.middle_initial))
+        self.__director_sheet.write(
+            self.__director_sheet_row_index, 3, format_non_date(director.last_name))
+        self.__director_sheet.write(
+            self.__director_sheet_row_index, 4, format_non_date(director.title))
+        self.__director_sheet.write(
+            self.__director_sheet_row_index, 5, format_date(director.appointment_date))
+        self.__director_sheet.write(
+            self.__director_sheet_row_index, 6, format_date(director.cessation_date))
+
+        delivery_address = director.delivery_address
```

Comment:
(BCorp) directors are getting more addresses. Ask @cameron-freshworks or @peter-freshworks about this.

Reply:
Multiple addresses for directors is already supported.